cmd/finch/nerdctl.go

```diff
@@ -474,3 +477,43 @@ var nerdctlCmds = map[string]string{
 	"volume":    "Manage volumes",
 	"wait":      "Block until one or more containers stop, then print their exit codes",
 }
+
+var managementCmds = []string{
+	"apparmor",
+	"builder",
+	"container",
+	"image",
+	"ipfs",
+	"namespace",
+	"network",
+	"system",
+	"volume",
```

Comment:
`apparmor`, `ipfs`, and `namespace` aren't in `nerdctlCmds` at [line 438 in this diff](https://github.com/runfinch/finch/pull/830/files#diff-403cda46333c3f741f8b1ff7f391901b0ccb8a5501cdd241138f89fd3c9db2c6R438).

If we add these, can we also regenerate the docs (run `./_output/bin/finch gen-docs generate -p ./docs/cmd` and commit the results)?